xml/System.Runtime.Versioning/RequiresPreviewFeaturesAttribute.xml

```diff
@@ -20,7 +20,7 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>To be added.</summary>
+    <summary>This attribute declares an API to be Preview and allows call sites to be flagged with a diagnostic indicating Preview API usage. Authors can use this attribute to ship Preview features in their assemblies.</summary>
     <remarks>To be added.</remarks>
```

Comment:
```suggestion
    <remarks>To be added.</remarks>
    <related type="ExternalDocumentation" href="https://aka.ms/dotnet-warnings/preview-features">Preview Features</related>
```

Reply:
Note: The link is not active yet, but will be by the time we ship.

Reply:
Change to `type="Article"`? Also, who is writing the documentation?

Reply:
@pgovind @tannergooding are you going to take care of this document? It should be up before we ship this.

Reply:
I've added the preview features properties to the MSBuild reference: https://docs.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props#enablepreviewfeatures. But I'll also plan to write a conceptual article.